docs/docs/app/datasources.mdx

```diff
@@ -8,63 +8,68 @@ slug: datasources
+## Overview
 
-You can use GrowthBook without a Data Source, but the user experience is not as smooth since you must enter all data manually.
+One of the great advantages of Growthbook is that all of your data stays in your data warehouse and only the aggregated statistics reaches GrowthBook servers.
 
-Below are the currently supported Data Sources (links are to our detail guides):
+Data Sources are how GrowthBook connects to your data warehouse so that it can pull those aggregated statistics in order to compute metrics and experiment results (You do not need to set one if all you are going to use is Growthbook Feature Flagging). Each Data Source defines how to connect to your data, what version of SQL to use when querying your data, and can provide templates for what the SQL to connect to your Data Source should look like depending upon which event tracker software you use. GrowthBook works with your existing SQL data, no matter where it is located and no matter what shape or format it is in, whether you have a strongly normalized schema, a single “events” table with JSON fields, or something in between.
 
-- Redshift
-- Snowflake
-- [BigQuery](/guide/bigquery)
-- ClickHouse
-- AWS Athena
-- Postgres
-- MySQL/MariaDB
-- MsSQL/SQL Server
-- PrestoDB (and Trino)
-- Databricks
-- [Mixpanel](/guide/mixpanel)
-- Google Analytics
-
-## Configuration Settings
+## Event Trackers
 
-To effectively use GrowthBook, you'll need to tell us a little about the shape of your data.
+When adding a new Data Source in Growthbook from /datasources page we first guide you to select what event tracker software you use, or you can choose custom if you don't use any of the popular third party event trackers that we support. Telling us which event tracker you use, gives us an idea on the likely shape of your data. This will help us generate the correct sql to extract out the aggregated statistics from your site with as little modification on your end as possible. Here is Growthbook's current list of event trackers that we support with links for more details on how to set them up with GrowthBook:
 
-### SQL Sources
+- [Amplitude](/app/event-trackers/amplitude)
+- [CleverTap](/app/event-trackers/clevertap)
+- Firebase
+- Freshpaint
+- Fullstory
+- [Google Analytics 4 (BigQuery only)](/app/event-trackers/GA4-google-analytics)
+- Heap Analytics
+- Jitsu
+- Keen IO
+- [Matomo](/app/event-trackers/matomo)
+- [Mixpanel](/app/event-trackers/mixpanel)
+- MParticle
+- [RudderStack](/app/event-trackers/rudderstack)
+- Segment
+- Snowplow
 
-GrowthBook can work with your existing SQL data, no matter what shape or format it is in, whether you have a strongly normalized schema, a single “events” table with JSON fields, or something in between.
+If you do not use any of those you choose [Custom Data Source](/app/event-trackers/custom) and define some of the sql yourself.
 
-#### Predefined Schemas
+## Data Source Types
```

Comment:
I'm not necessarily happy with the section title.  I use it because that is what we call it within our app.  Perhaps "Data Warehouses" would be a better title for this and perhaps also a better label for the field within the app?

Reply:
I don't have a strong opinion, but I'd like us to start adopting "Metric Source" and "Experiment Assignment Source" for those respective tables in our user's data warehouses. I'm not sure whether renaming this concept helps or hurts that, to be honest. I'd lean towards changing it to Data Warehouse, but I'd want to make sure we update this in as many places in the marketing + app itself.